File: vimeovideoinfo.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
# Informationen über ein Video mittels vimeo-API auslesen
# Lizenz: MIT license
# Datum: 09.11.2012
# 

import urllib2
import simplejson

videoid = "52073296"
url = "http://vimeo.com/api/v2/video/" + videoid + ".json"
handle = urllib2.urlopen(url)
content = handle.read()
j = simplejson.loads(content)

for key in j[0]:
	print "{0}: {1}".format(key,j[0][key])

File: vimeovideoinfo.php
<?php

// Informationen über ein Video mittels vimeo-API auslesen
// Lizenz: MIT license
// Datum: 09.12.2012
// 

$id = "52073296";
$url = "http://vimeo.com/api/v2/video/". $id .".json";
$j = json_decode(file_get_contents($url));

foreach($j[0] as $key => $value) {
	echo $key . ": " . $value . "\n<br />";
}

?>
